packages/client/components/ReflectionCard/ReflectionCard.tsx

```diff
@@ -55,16 +83,29 @@ const getReadOnly = (
 }
 
 const ReflectionCard = (props: Props) => {
-  const {meeting, reflection, isClipped, stackCount, showReactji, dataCy} = props
-  const {meetingId, reactjis} = reflection
+  const {meeting, reflection, isClipped, isReadOnly, stackCount, showReactji, dataCy} = props
+  const {
+    id: reflectionId,
+    content,
+    promptId,
+    isViewerCreator,
+    meetingId,
+    reactjis,
+    inSpotlight
+  } = reflection
   const phaseType = meeting ? meeting.localPhase.phaseType : null
+  const isComplete = meeting?.localStage?.isComplete
   const phases = meeting ? meeting.phases : null
-  const {id: reflectionId, content, promptId, isViewerCreator} = reflection
   const atmosphere = useAtmosphere()
   const {onCompleted, submitting, submitMutation, error, onError} = useMutationProps()
   const editorRef = useRef<HTMLTextAreaElement>(null)
   const [editorState, setEditorState] = useEditorState(content)
-
+  const [isHovering, setIsHovering] = useState(false)
+  const isDesktop = useBreakpoint(Breakpoint.SIDEBAR_LEFT)
```

Comment:
I find some of the breakpoints a bit confusing, e.g. why `SIDEBAR_LEFT` and `NEW_MEETING_SELECTOR` instead of say XS, S, M?

Reply:
fair point!
there are 2 schools of thought:
- bins by device size (S,M, L or maybe phone, tablet, desktop, TV)
- bins by functionality (hide the left nav, show 2 columns instead of 3, etc)

binning by device size was originally really popular, but becomes a headache if the sizes of devices vary widely. you're basically building your app to a set number of devices. it's still pretty popular for UI frameworks!

binning by functionality means the layout is determined by your components, not the hardware specs of your users. at least that's the thought.

that said, we've done a real bad job of sticking with 1 approach :see_no_evil:

Reply:
Makes sense - appreciate the explanation!